src/components/crossDevice/MobileSuccess/assets/return-to-computer-large.svg

```diff
@@ -0,0 +1,35 @@
+<svg width="144" height="144" viewBox="0 0 144 144" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Can you compress these icons?

Reply:
This one has already been optimised

Reply:
I ran it on https://jakearchibald.github.io/svgomg/ and it said it could reduce it by 16%

Reply:
Yes, but that will reduce the precision... It must have been because I've set precision to maximum.